src/extension.ts

```diff
@@ -672,11 +691,20 @@ function continueEditing(outline: DocumentOutline, useContext: boolean, selectio
 			);
 		}).then(offsetDiff => {
 			completion.translateAnchorOffset(offsetDiff);
-			return completion.completeText(
-				chatMessageBuilder.toChatMessages(),
-				userEndLineEol,
-				chatMessageBuilder.getCopilotOptions(),
-			);
+			if (errorMessage !== null) {
+				// Show errors to help users correct their text
+				completion.insertText(
+					errorMessage,
+					userEndLineEol
+				);
+			} else {
+				completion.completeText(
+					chatMessageBuilder.toChatMessages(),
+					userEndLineEol,
+					chatMessageBuilder.getCopilotOptions(),
+				);
+			}
+
 		}).catch(async error => {
 			// remove head error code
```

Comment:
This could be the culprit. We need to return promise of insertText|completeText.

Reply:
Fixed.